flake.nix

```diff
@@ -28,6 +28,8 @@
     mission-control.url = "github:Platonic-Systems/mission-control";
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
     nixpkgs-stable.url = "github:NixOS/nixpkgs/nixos-22.11";
+    pre-commit-hooks-nix.url = "github:hercules-ci/pre-commit-hooks.nix/flakeModule";
```

Comment:
This should be done via https://flake.parts instead

Reply:
It's used like this via flake-parts https://flake.parts/options/pre-commit-hooks-nix.html#installation